app/controllers/register_controller.rb

```diff
@@ -0,0 +1,29 @@
+class RegisterController < ApplicationController
+
+  def new
+  end
+
+  def create
```

Comment:
Feels weird to have User.new in the register controller but when we tried to put it in Users_controller it produced a template error.